config/schemas/serverSchema.js

```diff
@@ -0,0 +1,14 @@
+import * as z from "zod";
+
+const envSchema = z.object({
+  LINKFREE_MONGO_CONNECTION_STRING: z.string().min(10),
+  NODE_ENV: z
+    .enum(["development", "production", "test"])
+    .default("development"),
+  GITHUB_ID: z.string().optional(),
+  GITHUB_SECRET: z.string().optional(),
+  NEXTAUTH_SECRET: z.string().optional(),
+  LINKFREE_API_SECRET: z.string().min(1),
```

Comment:
I wonder if we make this a minimum of say `6`?

Reply:
is there any reason for it? I mean, I can make it require at least 6 chars. But if it's less than that, it'll throw an error.

Reply:
I think it would be better to encourage a longer number than 1 character